Node 22 as LTS, main and recommended Volto version (#6371)
plone/volto: docs/source/upgrade-guide/index.md

## Upgrading to Volto 18.x.x

### Node.js version support: adding 22, dropping 18

> Added support for Node.js 22, since it will become LTS on 2024-10-29.

Reviewer: ```suggestion
Added support for Node.js 22, since it is the new long-term support (LTS) release starting 2024-10-29.
```

> Long Term Support (LTS) for {term}`Node.js` 18 by the Node.js community will end on 2025-04-30.

Reviewer: ```suggestion
Long-term support for {term}`Node.js` 18 by the Node.js community will end on 2025-04-30.
```

Reviewer 2: This is a branding thing by Node.js, so "Long Term Support (LTS)" in this context is correct.

Reviewer: @stevepiercy Is it? https://nodejs.org/en/about/previous-releases says "long-term support" just like I did.

Reviewer 2: They use both: https://github.com/nodejs/release?tab=readme-ov-file#release-phases

Reviewer: Then I'd like to pick the one that seems more correct grammatically, which is "long-term support." https://www.grammarly.com/blog/punctuation-capitalization/hyphen-with-compound-modifiers/

Reviewer 2: Actually, it depends on context. I read it again, and you're correct, because it's a kind of support being offered, not a branded version being offered.

> Volto 18 no longer supports Node.js 18, since it only supports the last two LTS versions.

Reviewer 2: ```suggestion
Volto 18 no longer supports Node.js 18, since Volto only supports the latest two LTS versions.
```
